src/ui/app_view.rs

```diff
@@ -57,12 +60,13 @@ pub fn app_view(
 	let sidebar_scrolled = create_rw_signal(false);
 	let main_scroll_to = create_rw_signal(0.0);
 
-	let tooltip_signals_settings = TooltipSignals::new(que);
+	let que_settings = Que::default();
+	let tooltip_signals_settings = TooltipSignals::new(que_settings);
 	let overflow_labels = create_rw_signal(vec![0]);
 
 	let field_presets = create_rw_signal(env.config.get_field_presets());
 
-	let clear_icon = include_str!("./icons/clear.svg");
+	let delete_icon = include_str!("./icons/delete.svg");
```

Comment:
We had two icons that looked the exact same... removed the clear icon and just re-used the one we had